### ISSUE TITLE
mac encoding error

### ISSUE THREAD

**Issue by SPlanzer** [linz/linz-data-importer#48]
Below error reported by user

In trying to load the plug-in into QGIS (v3.4.4) running on my Mac (v10.13.6) the error shown below is produced. It is a showstopper.

```

Couldn't load plugin 'linz-data-importer' due to an error when calling its initGui() method

UnicodeDecodeError: 'ascii' codec can't decode byte 0xc5 in position 607: ordinal not in range(128)

Traceback (most recent call last):
  File "/Applications/QGIS3.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 344, in startPlugin
    plugins[packageName].initGui()
  File "/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/linz-data-importer/linz_data_importer.py", line 285, in initGui
    about_html=file.read()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc5 in position 607: ordinal not in range(128)
Python version: 3.6.6 (v3.6.6:4cf1f54eb7, Jun 26 2018, 17:02:57) [GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] QGIS version: 3.4.4-Madeira Madeira, exported

Python Path:

/Applications/QGIS3.app/Contents/MacOS/../Resources/python
/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python
/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins
/Applications/QGIS3.app/Contents/MacOS/../Resources/python/plugins
/Library/Frameworks/SQLite3.framework/Versions/D/Python/3.6
/Library/Frameworks/Python.framework/Versions/3.6/lib/python36.zip
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/lib-dynload
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python
/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins
/Users/petersharris/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/mmqgis/forms
```

The error is related to reading macrons in the about.html doc
```    about_html=file.read() ```

This is only related to osx. It has not be reported by the many linux and windows users (it would be, does not let the user use the plugin)

**Solutions:** 
* have an alternative doc without macrons that is read when this fails
  * But what about macrons in the capacities doc? 
* Review encoding (this was changed with the switch to python3).

**Comment by SPlanzer** [comment]
still relevant

**Comment by SPlanzer** [comment]
closed by #50 

QGIS in mac is defaulting to ASCII even though it is using python3 that should default to utf-8. Python3 in the mac terminal is using utf-8

```
locale.getprefferedencoding(False)
```

Tests not updated as there is a test for this but it only affects osx. Only thing that could be done is build mac CI
